UITextFieldPickerInput/PickerInputView.swift

```diff
@@ -15,6 +15,13 @@ public class PickerInputView: UIPickerView {
     private(set) var onSelect: SelectHandler?
     private(set) var onDone: SelectHandler?
     private(set) var selectedValue: [Int: Selectable] = [:]
+    private(set) var overlayedView: UIView?
+    
+    private lazy var tapGesture: UITapGestureRecognizer = {
+        let gesture = UITapGestureRecognizer(target: self, action: #selector(onPressOverlay))
```

Comment:
I believe you don't have to use a block here.
`private lazy var tapGesture = UITapGestureRecognizer(target: self, action: #selector(onPressOverlay))`
This should be enough